core/src/test/java/io/confluent/kafka/schemaregistry/avro/AvroCompatibilityTest.java

```diff
@@ -85,7 +157,7 @@ public void testBasicBackwardsCompatibility() {
                checker.isCompatible(schema2, Collections.singletonList(schema1)));
     assertFalse("adding a field w/o default is not a backward compatible change",
                 checker.isCompatible(schema3, Collections.singletonList(schema1)));
-    assertFalse("changing field name is not a backward compatible change",
+    assertTrue("changing field name with a proper alias is a backward compatible change",
```

Comment:
@junrao Do you remember the reasoning behind this test? I know aliases can be dangerous since you can create a series of compatible changes but not be able to project between two of them, i.e. the compatibility is not transitive. Was that the reason for marking changes that are technically compatible in Avro as incompatible in our API? Or was this just a mistake?